app/models/pglogical_subscription.rb

```diff
@@ -22,12 +22,15 @@ def self.find(*args)
     end
   end
 
-  def self.find_by_id(to_find)
+  def self.lookup_by_id(to_find)
     find(to_find)
```

Comment:
```suggestion
    find_by(:id => to_find)
```

wonder why we use `find` which raises the exception to then ignore the exception

Reply:
This is a defined find method in line 17, not the rails find method, maybe we should keep this find? Or rename the defined find method to lookup?

Reply:
aah - I see it now, it is really `find_id` - lets leave as is.

works best to keep move, refactor, and change separate.